### DISCUSSION TITLE
Docker Container Timezone Settings

### DISCUSSION THREAD

**Discussion by whitepiratebaku** [keycloak/keycloak#41843] [Q&A]
### Description

I have tried different ways to set container timezone, used setting TZ  env variable and also mounting host localtime to container but did not worked. Is there any way to set container time without recreating custom image?

**Comment by shawkins** [comment]
I don't think there is a way to do this without a custom image. From the 9.3 ubi micro release notes:

https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/9.3_release_notes/new-features#new-features-containers

UBI 9 Micro Container Image no longer contains zoneinfo installed by tzdata

With this update, the time zone information provided by the tzdata package is no longer included in UBI 9 Micro container images, consequently reducing the image size. The UBI 9 Minimal and UBI 9 Micro containers are UTC-only, and users should reinstall the tzdata package to get the full zoneinfo, if needed.

[Bugzilla:2223028](https://bugzilla.redhat.com/show_bug.cgi?id=2223028)
